cores/esp8266/core_esp8266_app_entry_noextra4k.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+ *  This is the original app_entry() not providing extra 4K heap, but allowing
+ *  the use of WPS.
+ *
+ *  see comments in core_esp8266_main.cpp's app_entry()
+ *
+ */
+
+#include <c_types.h>
+#include "cont.h"
+#include "coredecls.h"
+
+// calls to this function must *always* be inlined
```

Comment:
must *never* be inlined?

Reply:
I was meaning: `callers to this function must *always* be inlined`